editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -154,14 +154,33 @@ void HelpBrowser::createActions() {
     // For the sake of display:
     mWebView->pageAction(QWebEnginePage::Copy)->setShortcut(QKeySequence::Copy);
     mWebView->pageAction(QWebEnginePage::Paste)->setShortcut(QKeySequence::Paste);
+
+    // proxy page actions to avoid shortcuts conflicts with main window
+    action = mWebView->pageAction(QWebEnginePage::Back);
+    mActions[Back] = ovrAction = new OverridingAction(action->icon(), action->text(), this);
+    connect(ovrAction, SIGNAL(triggered()), action, SLOT(trigger()));
+    action->setShortcut(QKeySequence());
```

Comment:
maybe add a comment that the actual shortcut will be set via the `applySettings` method?